index.js

```diff
@@ -1,25 +1,54 @@
 'use strict';
-
+const Joi = require('joi');
```

Comment:
Let's create a new issue proposing the use of `joi` to validate the server extensions. This may already be built it, we should test it out.